### DISCUSSION TITLE
Live Mode - automatically turning on

### DISCUSSION THREAD

**Discussion by NawarA** [codesandbox/codesandbox-client#4069] [General]
# 🐛 bug report
This has been happening for several releases, but whenever I load any of my sandboxes, the editor automatically starts recording a Live session. 

## Description of the problem
Live sessions automatically turn on. Previously, Live Sessions would only turn on if a user chose to kick off a Live Session. As of a few releases ago, Live Sessions began turning on, without user input. This feels like you're always being monitored and recorded during your coding sessions.

## How has this issue affected you? What are you trying to accomplish?
It feels like Codesandbox is monitoring my sessions above and beyond whats required to provide the codesandbox service. Why are you recording a user session without the user expliclity kicking off live mode? If Live Mode is supposed to be enabled by default, and on all the time, remove it from the left hand menu. It feels like you're forcing live mode on everyone.

The take away is:
* Bug: Live mode is accidently enabled by default, as a few releases ago
* Bug: Your team decided Live Mode should be on 100% of the time, unless a user disables live mode; in which case, it feels like a bad user experience, since it feels like a user is being monitored without consent or choice. Which is a bug in UX, and a bug in your brand.

Either way, please patch "live mode" so its not enabled by default; or if you've chosen to make live mode on by default as a design choice, then please release a setting so I can opt-out of live mode; or finally, if you've chosen to give users zero choice on live mode, and you want it always on by default, then remove the Live Mode menu button, so users dont feel like they have a feature that's monitoring them without choice.

### Your Environment

| Software         | Name/Version |
| ---------------- | ------------ |
| Сodesandbox      | ec9e6ba33 |
| Browser          | Chrome 81 |
| Operating System | Windows 10 |

**Comment by MichaelDeBoey** [comment]
Hi @NawarA!

Since the latest release, Live Mode is enabled by default as you could see in the latest blogpost: https://codesandbox.io/post/redefining-code-collaboration#live-by-default
> All sandboxes are now live by default. Making collaboration seamless is key to its efficacy, so it's no longer a mode you enable but something that's on all the time. This means team members can come together and work on sandboxes with ease, and guests can be invited to view or edit outside of a specific session.

The CodeSandbox team chose to make this an opt-out instead of an opt-in for the reasons mentioned in the blogpost, so if you want, you can always opt-out of Live Mode in the settings of the sandbox.

**Comment by SaraVieira** [comment]
Hi @NawarA 

I am part of the CodeSandbox team and I moved your issue to a discussion as it seems it is mostly a discussion about what is the best way to proceed.

As Michael said we decided to turn on live by default a couple of releases ago to remove some small inconsistencies and other things @CompuIves can explain better 

Even though the session is live it doesn't mean we record any more than we would in normal circumstances, the only difference is that in live what you type goes through websockets so if you open in another tab these two will be able to communicate and stay in sync but only the owner of the sandbox and people in his team(if the sandbox is part of a team) will be able to follow each other.

I wanna make clear that when you share a sandbox that is live by default with the normal link it WILL NOT allow people to join your session, nothing about that has changed, the link for that is completely different and it's the one on the left side menu and that's why the menu is still there .

Does this make sense? What do you think could be improved on to better explain it if it does?

Thank you for your feedback, it's important to see the impact our changes have on people's perception whether those perceptions be good or bad

As of today there is no way of opting out of live by default in all sandboxes, only individual

**Comment by ertugrul59** [comment]
I believe this an privacy issue, because it starts recording for every refresh of the page, i hope you will solve this issue.

**Reply by sannek** [reply]
Hi @ertugrul59,

As Sara explained above, we do not record anything any more or different than usual in a live session, and nobody can join your live session unless they specifically have the live link.

When you close your sandbox, anything we might have recorded (to allow for syncing between different browser windows) will get swept and deleted.

I do agree it would be more intuitive if - once you turn off live for a sandbox - it would remain off until you turn it  back on, and I will pass that on to the team to see if/when we can change that.
